game-to-flatpak

```diff
@@ -23,17 +23,20 @@
 local fg = require "lib.flatpak-game"
 
 function usage()
-	print("game-to-flatpak [--network] [--bundle] [--keep-build-dir] [--keep-repo-dir] [FILE...]")
+	print("game-to-flatpak [--network] [--bundle] [--repo=DIR] [--keep-build-dir] [--keep-repo-dir] [FILE...]")
 end
 
 local file = nil
 local options = {}
+
```

Comment:
Please. no white space changes without justification (no reason to change it here...)